### PR TITLE
Added possibility to remove meter from the switch

### PR DESCRIPTION
**Pull request by nikitacherevko**

### CODE REVIEW

**Comment by nikitacherevko** [comment]
Need to test it on switches.

File: services/src/northbound/src/main/java/org/openkilda/northbound/controller/SwitchController.java [hunk 1]
```diff
@@ -235,4 +237,15 @@ public RulesValidationResult validateRules(@PathVariable(name = "switch_id") Str
     public RulesSyncResult syncRules(@PathVariable(name = "switch_id") String switchId) {
         return switchService.syncRules(switchId);
     }
+
+    /**
+     * Removes meter from the switch.
+     */
+    @ApiOperation(value = "Delete meter from the switch", response = DeleteMeterResult.class)
+    @GetMapping(path = "/switches/{switch_id}/meter/{meter_id}")
```
Comment (rtretyak) [inline services/src/northbound/src/main/java/org/openkilda/northbound/controller/SwitchController.java]: Should be DELETE method

**Comment by rtretyak** [comment]
Not working on staging, I am not receiving response back from northbound, it just hangs and then times-out. Wasn't able to retrieve any valuable info from kibana logs.

**Comment by rtretyak** [comment]
We should also consider whether we need endpoints for getting meters by id, getting all meters and bulk deleting all meters (maybe as a separate task or don't need at all)

**Comment by nikitacherevko** [comment]
Closes #952

File: services/src/northbound/src/main/java/org/openkilda/northbound/service/impl/SwitchServiceImpl.java [hunk 2]
```diff
@@ -243,4 +246,16 @@ public RulesSyncResult syncRules(String switchId) {
         return switchMapper.toRulesSyncResult(validationResult, syncResponse.getInstalledRules());
     }
 
+    @Override
+    public DeleteMeterResult deleteMeter(String switchId, long meterId) {
+        final String requestId = RequestCorrelationId.getId();
```
Comment (abochkarev) [inline services/src/northbound/src/main/java/org/openkilda/northbound/service/impl/SwitchServiceImpl.java]: What is the benefit of using final with variables here?

File: services/src/northbound/src/main/java/org/openkilda/northbound/service/SwitchService.java [hunk 3]
```diff
@@ -86,4 +87,11 @@
      * @return the synchronization result.
      */
     RulesSyncResult syncRules(String switchId);
+
+    /**
+     * Removed meter from the switch.
```
Comment (abochkarev) [inline services/src/northbound/src/main/java/org/openkilda/northbound/service/SwitchService.java]: Removed -> Removes. Could you please also divide the function describing section and params with \n?

File: services/src/northbound/src/main/java/org/openkilda/northbound/controller/SwitchController.java [hunk 4]
```diff
@@ -235,4 +237,15 @@ public RulesValidationResult validateRules(@PathVariable(name = "switch_id") Str
     public RulesSyncResult syncRules(@PathVariable(name = "switch_id") String switchId) {
         return switchService.syncRules(switchId);
     }
+
+    /**
+     * Removes meter from the switch.
+     */
```
Comment (abochkarev) [inline services/src/northbound/src/main/java/org/openkilda/northbound/controller/SwitchController.java]: Will it be possible to add the params section here?

**Pull request merged**